Library/Formula/bitlbee.rb

```diff
@@ -1,9 +1,9 @@
-require 'formula'
-
 class Bitlbee < Formula
   homepage 'http://www.bitlbee.org/'
-  url 'http://get.bitlbee.org/src/bitlbee-3.2.2.tar.gz'
-  sha1 '7e3cfe2b6bf4e8e603c74e7587307a6f5d267e9c'
+  url 'http://get.bitlbee.org/src/bitlbee-3.4.tar.gz'
```

Comment:
Double quotes everywhere here would be nice, since we're changing the lines anyway.